client/src/helper/localStorage.js

```diff
@@ -0,0 +1,35 @@
+export function getFromStorage(key) {
```

Comment:
having a helper file like this is pretty clever! great work

Reply:
thank you.